src/api/spec/services/notification_creator_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe NotificationCreator do
+  let(:user_bob) { create(:confirmed_user, login: 'bob') }
+  let(:user_kim) { create(:confirmed_user, login: 'kim') }
+  let(:commenter) { create(:confirmed_user, login: 'ann') }
+
+  let!(:bob_subscription) { create(:event_subscription_comment_for_project, receiver_role: 'maintainer', user: user_bob) }
+  let!(:kim_subscription) { create(:event_subscription_comment_for_project, receiver_role: 'maintainer', user: user_kim) }
+
+  let(:project) { create(:project, name: 'bobkim_project') }
+  let!(:relationship_bob) { create(:relationship_project_user, user: user_bob, project: project) }
+  let!(:relationship_kim) { create(:relationship_project_user, user: user_kim, project: project) }
+
+  let!(:comment_for_project) { create(:comment_project, commentable: project, user: commenter, body: 'blah') }
+  let(:event) { Event::Base.where(eventtype: 'Event::CommentForProject').last }
+  let(:comment_notifications) { Notification.where(notifiable_type: 'Comment') }
+
+  describe '#call' do
+    context 'without previous notification' do
+      before do
+        NotificationCreator.new(event).call
+      end
+
+      it 'two CommentForProject notification is created' do
+        expect(Notification::RssFeedItem.where(event_type: 'Event::CommentForProject').count).to eq(2)
+        expect(Notification::RssFeedItem.where(event_type: 'Event::CommentForProject').pluck(:subscriber_id)).to match_array([user_bob.id, user_kim.id])
+      end
+    end
+
+    context 'when the notification already exists' do
+      it 'does not create a duplicated notification' do
+        expect { NotificationCreator.new(event).call }.to change(Notification::RssFeedItem, :count).from(0).to(2)
```

Comment:
And this test is the `two CommentForProject notification is created`

Reply:
Refactor this tests. Can you have another look, please?